docs/docs/platform/workflows/index.md

```diff
@@ -0,0 +1,110 @@
+
+:::note
+When only name of workflow is changed and workflow is updated, identifier is not updated. It has to be updated separately.
+:::
+
+- **Workflow Group**
+
+The workflow group is used to group multiple workflows into a single group, currently only used behind the scenes for organizational purposes. But, in the upcoming subscriber preferences management, it will be used to group multiple notifications categories for the subscriber.
+
+- **Workflow Description**
+
+Workflow description is a field that is used to provides a concise and informative summary of the workflow. It highlights key features, specifications, and benefits, assisting other users in understanding the workflow's purpose.
+
+### Channel Settings
+
+![Workflow Channel Settings](/img/platform/workflow/channel-settings.png)
+
+For every workflow, all channels are enabled by default. Subscribers also have all channels enabled in their channel prefererence. Later, subscribers can change their own preference for a particualr template. **Users will be able to manage subscriptions** toggle option makes a template critical, if enabled.
+Read more on subscriber preferences and critical template [here](../preferences.md)
+
+## Workflow steps
+
+The workflow steps are nodes. You can add content in `channels` and configure `actions` as per your requirements
+
+Novu supports two type of steps:-
+
+1. **Channels** (in-app, email, push, sms and chat)
+2. **Actions** (digest and delay)
+
+![Workflow Steps](/img/platform/workflow/steps.png)
+
+## Trigger
+
+After a workflow is created, a `workflowIdenfier` will be automatically generated for it. `workflowIdentifier` is used to trigger the workflow to subscribers or topics.
+
+<Tabs groupId="language" queryString>
+  <TabItem value="js" label="Node.js">
+
+```javascript
+import { Novu } from '@novu/node';
+
+const novu = new Novu('<NOVU_API_KEY>');
+
+await novu.trigger('<WORKFLOW_IDENTIFIER>', {
+  to: {
+    subscriberId: '111',
+    email: 'john.doe@domain.com',
+    firstName: 'John',
+    lastName: 'Doe',
+    phone: '+13603963366',
+  },
+  payload: {
+    customVariable: 'variableValue',
+    organization: {
+      logo: 'https://organisation.com/logo.png',
+    },
+  },
+});
+```
+
+  </TabItem>
+</Tabs>
+
+## Test Workflow
+
+After adding all steps, actions and configuring a workfow, it can be tested using `Run a Test` option. Click on the `Get Snippet` button and then `Run a Test` tab to sending a test notification.
```

Comment:
```suggestion
After adding all steps, actions and configuring a workflow, it can be tested using `Run a Test` option. Click on the `Get Snippet` button and then `Run a Test` tab to send a test notification.
```